app-resources/src/main/resources/json/views/paikkis-3D.json

```diff
@@ -0,0 +1,217 @@
+{
+  "name": "Paikkatietoikkuna",
+  "type": "DEFAULT",
+  "default": true,
+  "public": true,
+  "onlyUuid": true,
+  "oskari": {
+    "application": "full-map-3D",
+    "page": "index",
+    "development_prefix": "/applications/paikkatietoikkuna.fi"
+  },
+  "bundles": [
+    { "id" : "lang-overrides" },
+    {
+      "id": "mapfull",
+      "config" : {
+        "layers": [],
+        "forceProxy": true,
+        "mapOptions": {
+          "resolutions": [
+            9783.939620502539,
+            4891.969810251269,
+            2445.9849051256347,
+            1222.9924525628173,
+            611.4962262814087,
+            305.74811314070433,
+            152.87405657035217,
+            76.43702828517608,
+            38.21851414258804,
+            19.10925707129402,
+            9.55462853564701,
+            4.777314267823505,
+            2.3886571339117526,
+            1.1943285669558763,
+            0.5971642834779382,
+            0.2985821417389691,
+            0.05
+          ],
+          "srsName": "EPSG:3857",
+          "terrain": { "providerUrl":"https://beta-karttakuva.maanmittauslaitos.fi/hmap/" }
+        },
+        "plugins": [
+          { "id": "Oskari.mapframework.bundle.mapmodule.plugin.LayersPlugin" },
+          { "id": "Oskari.mapframework.mapmodule.WmsLayerPlugin" },
+          { "id": "Oskari.mapframework.mapmodule.ControlsPlugin" },
+          { "id": "Oskari.mapframework.mapmodule.MarkersPlugin" },
+          {
+            "id": "Oskari.mapframework.mapmodule.GetInfoPlugin",
+            "config": { "infoBox": false }
```

Comment:
There's no mention in http://www.oskari.org/api/bundles#unreleased/mapping/mapmodule what the infobox: false does. Any idea?